css-three-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  background-color: #1a96d9;
+}
+
+.container {
+  max-width: 850px;
+  margin: 0 auto;
+}
+
+header {
+  width: 100%;
+  font-family: Lato, sans-serif;
+  font-size: larger;
+  color: white;
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 25px;
+}
+
+.full-column {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+  text-align: center;
+  margin: 0 20px;
+}
+
+div {
+  max-width: 100%;
+}
+
+img {
+  width: 25%;
+  border-radius: 50%;
+}
```

Comment:
You're immediately overriding these styles right below this on lines 45 - 51. If none of these styles are being applied, why include them?